src/handshake/machine.rs

```diff
@@ -101,6 +103,7 @@ pub enum RoundResult<Obj, Stream> {
 #[derive(Debug)]
 pub enum StageResult<Obj, Stream> {
     /// Reading round finished.
+    #[allow(missing_docs)]
```

Comment:
Hm, why was this required? The `DoneReading` is documented...

Reply:
I don't know. but without it, an error will occure.